wpiutil/src/main/native/thirdparty/llvm/include/wpi/MathExtras.h

```diff
@@ -191,10 +216,12 @@ inline int64_t minIntN(int64_t N) {
 
 /// Gets the maximum value for a N-bit signed integer.
 inline int64_t maxIntN(int64_t N) {
-  assert(N > 0 && N <= 64 && "integer width out of range");
+  assert(N <= 64 && "integer width out of range");
```

Comment:
This needs the `> 0` check, as N is signed.

Reply:
`>= 0` should be sufficient since the function checks for 0.